### ISSUE TITLE
[RFC] Moving Tekton to the CNCF

### ISSUE THREAD

**Issue by jerop** [tektoncd/community#1080]
tl;dr: We are proposing moving Tekton to the CNCF.

Tekton has been a great success as a CDF graduated project, and we are grateful for the support from the community and the CDF. 

We believe that the CNCF will be the right home for Tekton as it continues to grow and evolve. The CNCF has a strong track record of success in incubating and fostering open source projects, and we're confident that they will be able to provide Tekton with the support it needs to reach its full potential.

We are also excited about the opportunities that this move will create for collaboration with other CNCF projects. Tekton is a natural fit for the CNCF ecosystem, for example Tekton is built on Kubernetes and Tekton resources are distributed in Artifact Hub. We’re looking forward to working with other projects to build a better future for cloud native computing.

We are looking for feedback from the community! We welcome any questions and feedback on this thread, and will host a community meeting on October 11th to discuss more face-to-face.

Tekton Governing Board
@tektoncd/governing-board

**Comment by JaimeMagiera** [comment]
There are a lot of benefits, including increases visibility, which translates to increased usage and increased contributors. I can't think of any downsides.

**Comment by jerop** [comment]
After the community discussions, we have started working on the application to join the CNCF. We will update this issue with a link to the application when we submit it.

**Comment by aThorp96** [comment]
For posterity, here is the submitted CNCF application: https://github.com/cncf/toc/issues/1310

**Comment by aThorp96** [comment]
The Application has moved through the Due Diligence, Adopter Interviews, and TOC internal comment period. An acceptance vote was called on March 10 and is in progress: https://github.com/cncf/toc/issues/1310#issuecomment-4033459189

**Comment by aThorp96** [comment]
Tekton has been accepted as a CNCF incubating project! 
https://www.cncf.io/blog/2026/03/24/tekton-becomes-a-cncf-incubating-project/

@vdemeester I believe this issue can be closed 🙂
